addons/im_livechat/static/src/core/@types/models.d.ts

```diff
@@ -3,6 +3,8 @@ declare module "models" {
         livechat: DiscussAppCategory,
     }
     export interface Thread {
+        anonymous_country: Object,
+        anonymous_name: String,
```

Comment:
I'm not fan to mix purely channel data into thread model, but I understand how it allows progressing for now. We should create the Channel model inheriting Thread at some point later for this kind of stuff.

Reply:
These fields are poor because they relate to correspondent data. They should not be stored in formatter of thread but in formatter of correspondent.

If we do this, then it doesn't necessarily need to be specific to Channel, so not a reason to push for more differences between channels and chatter (which I strongly push for the code to be the same, differences come more from poor implementation on our part than them being conceptually different)